.github/renovate.json5

```
@@ -6,9 +6,16 @@
   packageRules: [
     {
       matchPackageNames: ['sigs.k8s.io/cluster-api', 'kubernetes-sigs/cluster-api'],
-      groupName: 'sigs.k8s.io/cluster-api',
-    },
+      groupName: 'sigs.k8s.io/cluster-api'
+    }
   ],
+  prFooter: "**Release note**:\n\
```

```suggestion
  prFooter: "**Release notes**:\n\
```

Is it singular in our other repos as well?

---

Oke 👍